.ci/setup_env_opensuse.sh

```diff
@@ -31,7 +31,7 @@ declare -A packages=( \
 	[os_tree]="libostree-devel" \
 	[libudev-dev]="libudev-devel" \
 	[metrics_dependencies]="smemstat jq" \
-	[cri-containerd_dependencies]="libseccomp-devel libapparmor-devel make pkg-config" \
+	[cri-containerd_dependencies]="libseccomp-devel libapparmor-devel make pkg-config libbtrfs-devel" \
```

Comment:
@lifupan can you try to add here `apparmor-profiles` and see if it gets better?

Reply:
Or even better `patterns-base-apparmor`

Reply:
Hi @marcov, thanks, it seems work after I added patterns-base-apparmor.

Reply:
Great, finally we can merge this.